cmake/scripts/common/Macros.cmake

```diff
@@ -499,7 +499,7 @@ macro(core_add_optional_subdirs_from_filelist pattern)
       list(GET line 1 subdir_dest)
       list(GET line 3 opts)
       foreach(opt ${opts})
-        if(ENABLE_${opt})
+        if(ENABLE_${opt} OR ${opt}_FOUND)
```

Comment:
Hm, can it be that optional dependencies didn't have ENABLE_ set? I'd just remove it and see if it builds.